test/container.spec.ts

```diff
@@ -0,0 +1,93 @@
+      await frame.type("input[class='inputToken']", connStr)
+      await frame.click("input[value='Connect']")
+
+      // create new collection
+      await frame.waitFor('button[data-test="New Container"]', { visible: true })
+      await frame.waitForSelector('div[class="splashScreen"] > div[class="title"]', { visible: true })
+      await frame.click('button[data-test="New Container"]')
+      
+      // check new database
+      await frame.waitFor('input[data-test="addCollection-createNewDatabase"]')
+      await frame.click('input[data-test="addCollection-createNewDatabase"]')
+
+      // check shared throughput
+      await frame.waitFor('input[data-test="addCollectionPane-databaseSharedThroughput"]')
+      await frame.click('input[data-test="addCollectionPane-databaseSharedThroughput"]')      
+
+      // type database id
+      await frame.waitFor('input[data-test="addCollection-newDatabaseId"]')
+      await frame.type('input[data-test="addCollection-newDatabaseId"]', dbId)      
+
+      // type collection id
+      await frame.waitFor('input[data-test="addCollection-collectionId"]')
+      await frame.type('input[data-test="addCollection-collectionId"]', collectionId)      
+
+      // type partition key value
+      await frame.waitFor('input[data-test="addCollection-partitionKeyValue"]')
+      await frame.type('input[data-test="addCollection-partitionKeyValue"]', sharedKey)
+
+      // click submit
+      await frame.waitFor('#submitBtnAddCollection')
+      await frame.click('#submitBtnAddCollection')
+
+      // validate created
+      // open database menu
+      await frame.waitFor(`span[title="${dbId}"]`)
+      await frame.waitForSelector('div[class="splashScreen"] > div[class="title"]', { visible: true })
+
+      await frame.click(`div[data-test=${dbId}]`)
+      await frame.waitFor(`span[title="${collectionId}"]`)
+      
+      // delete container
+
+      // click context menu for container
+      await frame.waitFor(`div[data-test="${collectionId}"] > div > button`)
+      await frame.click(`div[data-test="${collectionId}"] > div > button`)
+
+      // click delete container
+      const msLayer = await frame.waitForSelector('body > div.ms-Layer.ms-Layer--fixed')
+      await msLayer.$$eval('span[class="treeComponentMenuItemLabel"]', elementArray => {
+        // we cast an Element as HTML since it is clickable, but TS seems to think it's not
+        return (elementArray[4] as HTMLElement).click()
+      })
+
+      // confirm delete container
+      await frame.type('input[data-test="confirmCollectionId"]', collectionId.trim())
+
+      // click delete
+      await frame.click('input[data-test="deleteCollection"]')
+      await frame.waitForSelector('div[class="splashScreen"] > div[class="title"]', { visible: true })
+
+      await expect(page).not.toMatchElement(`div[data-test="${collectionId}"]`)
+    } catch (error) {
+      await page.screenshot({path: 'failure.png'});
```

Comment:
Nice. Can pupeteer specify its own screenshot filename (based on, say, test name) so that subsequent calls of `page.screenshot` don't override this file?

Reply:
i would think so, yeah. This is a bit of copypasta from another test, but can make that dynamic

Reply:
@languy I have a solution for this in another PR, planning to merge it with my addition of the cassandra test